packaging/installer/README.md

```diff
@@ -52,7 +52,7 @@ To learn more about the pros and cons of using *nightly* vs. *stable* releases,
 Verify the integrity of the script with this:
 
 ```bash
-[ "0ae8dd3c4c9b976c4342c9fc09d9afae" = "$(curl -Ss https://my-netdata.io/kickstart.sh | md5sum | cut -d ' ' -f 1)" ] && echo "OK, VALID" || echo "FAILED, INVALID"
+[ "7278ce1688b16dd3156613971726ba01" = "$(curl -Ss https://my-netdata.io/kickstart.sh | md5sum | cut -d ' ' -f 1)" ] && echo "OK, VALID" || echo "FAILED, INVALID"
```

Comment:
Because it's unfortunately still more ubiquitous than `sha1sum` (not that that's secure either, yet another attack (this time economically viable for small attackers) was discovered just recently).

Honestly though, I'm of the opinion we should be changing these oneline 'verification' commands, AFAICT they don't actually do what the original author seems to think they did, and we should be listing a wider variety of checksums including more secure options than MD5. This PR isn't really the place to deal with that though.